package.json

```diff
@@ -130,6 +130,7 @@
     "react-input-autosize": "1.1.0",
     "react-intl": "2.2.3",
     "react-joyride": "1.10.1",
+    "react-notification-system": "0.2.14",
```

Comment:
Yes, i don't like very much the implementation too. This is why I wrote the wrapper, that allows to use it in a one way data flow (as many of our existing components). 

Maybe in the future we can replace it modifying only the wrapper (the model for the notification is very simple, so it can be replaced quickly), or maybe they will improve the lib. 

But for the moment we have a notification system that works, without creating divs here and there to notify success, warning or errors. 

I've seen the wrapper you linked. It doesn't work as well as expected in our use case: 
- Uses the store in the context
- doesn't seems to work on mount, but only on update (probably the author didn't understand the difference between `componentDidUpdate` and `componentWillReceiveProps`).

For our plugin system this solution is better in terms of maintenance.